android/src/main/java/net/no_mad/tts/TextToSpeechModule.java

```diff
@@ -76,6 +76,7 @@ public class TextToSpeechModule extends ReactContextBaseJavaModule {
     private long audioGainMin = (long) (1.0 * UNITY_GAIN_Q8p24); // 0dB = 1.0 (10 ^ (0/20))
     private long largestSample = 0;
     private AudioTrack audioTrack;
+    private Float volume = 1.0f;
```

Comment:
Why not use the primitive type `float` here?

Reply:
Intention was indeed for it to be `float`, changed in commit [#f24d558](https://github.com/tomtom-international/react-native-tts/pull/27/commits/f24d5586a821664f032a77e2d9d8f9333fa6e01e)